ansible/roles/ocp4_workload_integreatly/defaults/main.yml

```diff
@@ -0,0 +1,59 @@
+---
+become_override: False
+ocp_username: opentlc-mgr
+silent: False
+
+# Operator
+ocp4_workload_integreatly_user_count: 50
+ocp4_workload_integreatly_tmp_dir: /tmp
+ocp4_workload_integreatly_prefix: redhat-rhmi-
+ocp4_workload_integreatly_namespace: "{{ ocp4_workload_integreatly_prefix }}operator"
+ocp4_workload_integreatly_release: "2.1.0"
+ocp4_workload_integreatly_git_ref: "v{{ ocp4_workload_integreatly_release }}"
+ocp4_workload_integreatly_manifest_dir_type: 'url'
+ocp4_workload_integreatly_manifest_dir_format: https://raw.githubusercontent.com/integr8ly/integreatly-operator/{branch}/deploy/olm-catalog/integreatly-operator
+ocp4_workload_integreatly_manifest_csv_formats:
+- "{release}/integreatly-operator.v{release}.clusterserviceversion.yaml"
+ocp4_workload_integreatly_manifest_package_formats:
+- integreatly-operator.package.yaml
+ocp4_workload_integreatly_manifest_crd_formats:
+- "{release}/integreatly.org_rhmis_crd.yaml"
+ocp4_workload_integreatly_custom_resource_name: integreatly
+ocp4_workload_integreatly_custom_resource_type: workshop
+ocp4_workload_integreatly_custom_resource_self_signed: false
+ocp4_workload_integreatly_custom_resource_cluster_storage: "true"
+ocp4_workload_integreatly_catalog_source_configmap_name: rhmi-registry-cm
+ocp4_workload_integreatly_catalog_source_name: rhmi-registry-cs
+ocp4_workload_integreatly_operator_group_name: rhmi-registry-og
+ocp4_workload_integreatly_subscription_approval: Automatic
+ocp4_workload_integreatly_subscription_group_package: integreatly
+ocp4_workload_integreatly_subscription_channel: rhmi
+
+# Users
+ocp4_workload_integreatly_dedicated_admin_username: dedicated-admin
+ocp4_workload_integreatly_user_password: password
+ocp4_workload_integreatly_dedicated_admin_user_password: password
+ocp4_workload_integreatly_admin_user_password: password
+
+# 3scale
+ocp4_workload_integreatly_threescale_namespace: redhat-rhmi-3scale
+ocp4_workload_integreatly_seed_secret_name: system-seed
+ocp4_workload_integreatly_threescale_tenant_admin_user: admin
+ocp4_workload_integreatly_threescale_tenant_admin_email: admin@example.com
+ocp4_workload_integreatly_threescale_tenant_admin_password: Password1
+ocp4_workload_integreatly_threescale_tenant_detail_secret_name: admin-tenant
+ocp4_workload_integreatly_threescale_tenant_detail_secret_namespace: redhat-rhmi-3scale
+ocp4_workload_integreatly_threescale_tenant_id_format: evals%02d-tenant
+ocp4_workload_integreatly_threescale_users_name_format: evals%02d
```

Comment:
been using this var for fuse as well, maybe better to move it to the operator section